addons/web/static/src/js/views/basic/basic_controller.js

```diff
@@ -559,7 +559,10 @@ var BasicController = AbstractController.extend(FieldManagerMixin, {
      * @param {string[]} ids list of deleted ids (basic model local handles)
      */
     _onDeletedRecords: function (ids) {
-        this.update({});
+        if (this._searchPanel)
+            this._searchPanel._notifyDomainUpdated();
+        else
+            this.update({});
```

Comment:
```suggestion
        if (this._searchPanel) {
            this._searchPanel._notifyDomainUpdated();
        } else {
            this.update({});
        }
```

To be compliant with the JS guidelines[0] :-)

[0] : https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines#generic-guidelines